web/src/test/java/com/microsoft/applicationinsights/web/extensibility/modules/WebUserTrackingTelemetryModuleTests.java

```diff
@@ -62,11 +66,17 @@
     @BeforeClass
     public static void classInitialize() throws Exception {
         server.start();
+
+        // Set mock channel
+        channel = MockTelemetryChannel.INSTANCE;
```

Comment:
I'm not sure that this change is related to the user/session cookies (i.e. try not to mix unrelated changes in a single commit)